README.md

```diff
@@ -1,5 +1,98 @@
+---
 
-[![Netlify Studio Status](https://api.netlify.com/api/v1/badges/4ea58d4f-a4db-4ed5-a899-c1fcbcd9f1f8/deploy-status)](https://app.netlify.com/sites/iflix-studio-develop/deploys)
+### **In one line**,
+
+iFlix is a platform to watch and create interactive movies by utilizing multiple module's offered by the platform.
+
+### **Who is this for?**
+
+| Persona     | Who are they?                                                                               | What is their main goal?                                                                                      |
+| :---------- | :------------------------------------------------------------------------------------------ | :------------------------------------------------------------------------------------------------------------ |
+| A Cinephile | A person with a passionate interest in cinema                                               | They want to be able to interact and be part of the story by altering it with their actions                   |
+| An Editor   | A film editor controls a film's artistic and dramatic aspects and visualizes the screenplay | They want to be able to add interactivity to movies where a cinephile will have the choice to alter the story |
+| An Admin    | A person who is responsible for the administrator tasks                                     | They want to be able to manage the movie lists, have access controls                                          |
+
+### **How do they Benefit?**
+
+iFlix is a platform where viewers can interact with the film and influence the events that unfold in the film.
+
+iFlix allows users to watch interactive movies from any and every device they own, with ease. A user can use the studio module to add interactivity to movies, manage, publish movies on the platform and get statistics for different movies like the number of views etc. As an iFlix admin, the user can perform essential admin tasks like managing movies, users, access rights etc.
+
+### **How is this different?**
+
+In a regular movie/cinematic experience, the viewer lacks the option to alter events unfolding within the film. In video games, the player is given the freedom to interact with the game and alter the storyline.
+Interactive movies are a combination of both cinematic experience and altering the storyline as per the user's choice. However, Interactive movie/cinema is a concept that has been around since the ’90s and has remained in the background because of lack of success.
+The goal of iFlix is to give the viewer the possibility to alter the story, while still enjoying a cinematic experience.
+
+---
+
+## **Live Link**
+
+- **[OTT Module](https://i-flix-develop.netlify.app/)** [![Netlify Production Status](https://api.netlify.com/api/v1/badges/01521a19-5b5e-4fae-9888-6ab2e0a1677f/deploy-status)](https://app.netlify.com/sites/i-flix/deploys)
```

Comment:
I think we should add deployed links rather than this dummy link here. Can we put deployed links of develop branch here so that viewers can review the live project?